src/lumigo_tracer/user_utils.py

```diff
@@ -110,6 +110,25 @@ def validate_tag(key, value, tags_len, should_log_errors):
     return True
 
 
+def manual_trace(func):
+    def wrapper(*args, **kwargs):
+        with ManualTrace(func.__name__):
+            func(*args, **kwargs)
+
+    return wrapper
+
+
+class ManualTrace:
```

Comment:
you can write this code better with https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager